mala/descriptors/bispectrum.py

```diff
@@ -175,13 +181,14 @@ def _calculate(self, atoms, outdir, grid_dimensions, **kwargs):
             # deallocates memory too quickly. This copy is more memory
             # hungry, and we might have to tackle this later on, but
             # for now it works.
-            return snap_descriptors_np.copy(), nrows_local
+            return snap_descriptors_np, nrows_local
```

Comment:
The comment above says that the `copy()` is needed, but this PR removes it.

Reply:
The copy has been moved in the lammps_util.py file, but you are right, the comment should be adapted. Will do that.